src/adapters/DebugProtocolAdapter.ts

```diff
@@ -376,6 +389,9 @@ export class DebugProtocolAdapter {
                         lastPartialLine = '';
                     }
                     // Emit the completed io string.
+                    this.processUnhandedOutputForDebugPrompt(responseText.trim()).catch((e) => {
```

Comment:
We don't need to .catch this. The whole flow can happen synchronously.

Reply:
Done